### PR TITLE
GOK-366 | Inward Api functionality for stock receipt

### PR DESCRIPTION
**Pull request by yenugukeerthana**
## Requirements

- [ ] This PR has a proper title that briefly describes the work done
- [ ] I have squashed / amended the comments to make it more redable
- [ ] I have included link to all the JIRA ticket('s)
- [ ] I wrote the code as a pair or atleast performed a self-review of my own code
- [ ] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/README.md) (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/inventory/aushada/aushada.jsx [hunk 1]
```diff
@@ -52,53 +60,99 @@ const Aushada = (props) => {
...
+  const [instituteIdExists, setInstituteIdExists] = useState(false);
+  const locationUuid = cookies[locationCookieName].uuid;
 
   const { data: stockRoom, error: stockRoomError } = useSWR(
     stockRoomURL(cookies[locationCookieName]?.name.trim()),
     fetcher,
   );
 
+  useEffect(async ()=>{
+    const response=await getRequest(globalPropertyUrl('eAushadha.inward'))
```
Comment (deepthi-mantena) [inline src/inventory/aushada/aushada.jsx]: Can we rename this variable name from response to somethoing which indicates the flag from globalproperty for inwardApi

File: src/inventory/aushada/aushada.jsx [hunk 2]
```diff
@@ -52,53 +60,99 @@ const Aushada = (props) => {
...
+    if(response==true){
```
Comment (deepthi-mantena) [inline src/inventory/aushada/aushada.jsx]: Here we can also avoid boolean comparision

File: src/inventory/aushada/aushada.jsx (hunk 1 above)
Comment (yenugukeerthana) [reply src/inventory/aushada/aushada.jsx]: Done

File: src/inventory/aushada/aushada.jsx (hunk 2 above)
Comment (yenugukeerthana) [reply src/inventory/aushada/aushada.jsx]: Done

**Review by deepthi-mantena — APPROVED** [review]

**Pull request merged**